src/azure-cli/azure/cli/command_modules/managedservices/_command_type.py

```diff
@@ -0,0 +1,23 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import AzCommandGroup
+
+
+class ManagedServicesCommandGroup(AzCommandGroup):
+    def managedservices_custom(self, name, method_name=None, **kwargs):
+        self._register_common_arguments(name, method_name, **kwargs)
+
+    def _register_common_arguments(self, name, method_name=None, **kwargs):
+        command = self.command_loader.command_table[self.custom_command(name, method_name, **kwargs)]
+
+        group_name = 'ManagedServices'
+        command.add_argument('api_version', required=False, default=None,
+                             arg_group=group_name, help='The API Version to target.')
+
+        if method_name in ("cli_assignment_list", "cli_assignment_get"):
+            from azure.cli.core.commands.parameters import get_three_state_flag
+            command.add_argument('include_definition', arg_group=group_name, arg_type=get_three_state_flag(),
+                                 help='When provided, gets the associated registration definition details.')
```

Comment:
The standard way to register this would be for `cli_assignment_list` and `cli_assignment_get` to both have `include_definition` in their signature and then have a simple `c.argument('include_definition', …)` line in params.py. Is there a reason you are doing it this way?

Reply:
Fixed.